src/App.js

```diff
@@ -8,8 +9,10 @@ const article = {
   date: 'August 30, 2019',
 };
 
-const App = () => (
-  <h1>React Article</h1>
-);
+const App = () => {
+  const { title, date, text } = article;
+
+  return <Article title={title} date={date} text={text} />;
```

Comment:
remember, what we do when we have more than 3 attributes for a tag? we put each o them on a new line
```suggestion
  return (
    <Article 
      title={title} 
      date={date}
      text={text}
    />
  );
```